### ISSUE TITLE
package_data does not copy files

### ISSUE THREAD

**Issue by cvanelteren** [pypa/setuptools#2446]
I have a simple setup file that compiles cython code. However, I want to package my `*.pxd` files for other cython packages to use. The `package_data` option does not copy the `pxd` files to the installed directory regardless of what options I use.

MANIFEST.in
```
glob-include *.pxd
global-exclude __pycache__
```

```python
[...]
setup(
...
package_dir = {"", "plexsim"},
package_data ={"", ["*pxd"]},
include_package_data = True
)
```

My sources file lists the to be packaged  files

[/...egg/SOURCES.TXT]
```
plexsim/models.cpp
plexsim/models.pxd
plexsim/models.pyx
```

However the installed directory remains empty 

[python3.8/site-packages]
```
models.cpython-38-x86_64-linux-gnu.so
models.py
__pycache__
```

The only work-around I have found is to use the `data_files` option, however this seems a bit odd as even in the docs it's stated as a 'else' option for packing data. 

Am I missing something with my setup file? Why is the added option not used?

Setuptools: 50.3.0.post20201006
python: 3.8.6

**Comment by abravalheri** [comment]
Hi @cvanelteren, have you tried this thing again with version 58.5.3?
Maybe the problem was solved with https://github.com/pypa/setuptools/pull/2844.

**Comment by abravalheri** [comment]
For the time being, I will close this issue since I believe it was solved by #2844.
But please feel free to re-open it (by adding a comment with more information regarding a more recent version of setuptools).
